python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -681,10 +681,13 @@ def _groupby_supported(gb):
     )
 
 
-def _make_name(*args, sep="_"):
-    """Combine elements of `args` into a new string"""
-    _args = (arg for arg in args if arg != "")
-    return sep.join(_args)
+def _make_name(col_name, sep="_"):
+    """Combine elements of `col_name` into a single string, or no-op if
+    `col_name` is already a string
+    """
+    if isinstance(col_name, str):
+        return col_name
+    return sep.join(name for name in col_name if name != "")
```

Comment:
I don’t think I understand why this function needs to be changed at all.  I do see why you need to call this function differently in some cases, but as far as I can tell, the behavior of the function itself is exactly the same as before (I realize I am probably misunderstanding).

Reply:
Eh - I see why you did this.

Feel free to ignore my comment above. I'm okay with the change :)